apps/native-component-list/src/screens/Video/VideoScreen.tsx

```diff
@@ -1,26 +1,53 @@
-import { VideoView, useVideoPlayer } from '@expo/video';
+import { useVideoPlayer, VideoView, VideoSource } from '@expo/video';
+import { Picker } from '@react-native-picker/picker';
+import { Platform } from 'expo-modules-core';
 import React, { useCallback, useEffect, useRef } from 'react';
-import { PixelRatio, ScrollView, StyleSheet, View, Text } from 'react-native';
+import { PixelRatio, ScrollView, StyleSheet, Text, View } from 'react-native';
 
 import Button from '../../components/Button';
 import TitledSwitch from '../../components/TitledSwitch';
 
+const bigBuckBunnySource: VideoSource =
+  'http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/BigBuckBunny.mp4';
```

Comment:
Does `http` work for you? It stopped working for me a while ago, see https://github.com/expo/expo/pull/27036
Let's use `https` if possible.

Reply:
Interesting, it works fine for me. I'll switch to https though